tools/DevDiagnostics/DevHome.DevDiagnostics/Helpers/WERUtils.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+using Microsoft.Win32;
+
+namespace DevHome.DevDiagnostics.Helpers;
+
+internal sealed class WERUtils
```

Comment:
No exception handling in this class, nor in its caller...?

Reply:
No, if it fails, something bad has happened... not much we can do to recover.

Reply:
Right, but log before we failfast?

Reply:
I'm not sure where/what I would log. If I'm unable to read the registry, the exception will bubble up to either DevDiagnostics or EnableLocalCabCollection and result in a Watson crash.

Same goes with the writing.... it would fail if this is called from a medium IL context, but should work elevated. If I'm wrong, there will be a Watson crash to track the failure.